Overview.src.html

```diff
@@ -1851,10 +1873,38 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   for HTTP/2, and equivalent information used to prioritize dispatch and processing of
   HTTP/1 fetches.
 
+
+ <li><p>If <var>request</var> is a <span>navigation request</span>, a user agent should <span title=concept-header-list-append>append</span> the following pairs to <var>request</var>'s
+   <span title=concept-request-header-list>header list</span>:
+   <ol>
+    <li>`<code title=http-dpr><a href="http://httpwg.org/http-extensions/client-hints.html#dpr">DPR</a></code>`/<var>value</var>, unless the <span title=concept-request-header-list>header list</span> already contains a <span title=concept-header>header</span> whose <span title=concept-header-name>name</span> is `<code title=http-dpr>DPR</code>`
+
+    <li>`<code title=http-save-data><a href="http://httpwg.org/http-extensions/client-hints.html#save-data">Save-Data</a></code>`/<var>value</var>, unless the <span title=concept-request-header-list>header list</span> already contains a <span title=concept-header>header</span> whose <span title=concept-header-name>name</span> is `<code title=http-save-data>Save-Data</code>`
+    <li>`<code title=http-viewport-width><a href="http://httpwg.org/http-extensions/client-hints.html#save-data">Viewport-Width</a></code>`/<var>value, unless the <span title=concept-request-header-list>header list</span> already contains a <span title=concept-header>header</span> whose <span title=concept-header-name>name</span> is `<code title=http-viewport-width>Viewport-Width</code>`</var>
+  </ol>
+
+  <p>where <var>value</var> is set according to the definition in <span data-anolis-ref>CLIENT-HINTS</span>.
```

Comment:
I don't think this makes much sense since this means value would be the same for all these headers.